tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<ul><li> "If the weight is null then no light is sent on the atom": I think mentioning light being sent is confusing, because it is refering to the inner workings of the DMM which we don't explain. Instead, we can mention that no detuning from the DMM is applied or something along those lines </li><li>"in a real <code>Device</code> the trap layout has to be calibrated and only some layouts can be used, defined in <code>calibrated_layouts</code> of the <code>Device</code>": not strictly true, you can use registers with other layouts and even register without layouts with <code>Device</code> to. It's only execution on a QPU that's blocked by this. I would say maybe we can just not say this </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>About the second point: Now that I am re-reading myself I agree this sentence is useless haha</p>